src/Cache/CacheClient.php

```diff
@@ -1168,4 +1170,104 @@ private function getNextDataClient(): ScsDataClient
         $this->nextDataClientIndex = ($this->nextDataClientIndex + 1) % count($this->dataClients);
         return $client;
     }
+
+    /**
+     * Gets the cache values stored for given keys.
+     *
+     * @param string $cacheName Name of the cache to perform the lookup in.
+     * @param array $keys The keys to look up.
+     * @return ResponseFuture<GetBatchResponse> A waitable future which will provide
+     * the result of the get operations upon a blocking call to wait.
+     * <code>$response = $responseFuture->wait();
+     * }</code>
+     * The response represents the result of the get operation and stores the
+     * retrieved value. This result is resolved to a type-safe object of one of
+     * the following types:<br>
+     * * GetBatchSuccess<br>
+     * * GetBatchError<br>
+     * Pattern matching can be to operate on the appropriate subtype:<br>
+     * <code>if ($success = $response->asSuccess()) {<br>
+     * &nbsp;&nbsp;$value = $success->getResponses();<br>
```

Comment:
This should be either `$results = $success->results();` or `$values = $success->values();` (or maybe both), but I would like to double check with @cprice404 that that's ok for the final naming. Actually, the docs we have for sources of truth for naming here aren't exactly clear, so we should have a quick discussion about the desired outcome and then feed that back into the docs.